raiden_contracts/tests/test_channel_settle.py

```diff
@@ -85,124 +79,72 @@ def test_settle_channel_state(
         withdraw_channel,
         close_and_update_channel,
         settle_state_tests,
-        channel_test_values,
 ):
-    number_of_channels = 5
-    accounts = get_accounts(2 * number_of_channels)
-    (vals_A0, vals_B0) = channel_test_values
-
-    # We mimic old balance proofs here, with a high locked amount and lower transferred amount
-    # We expect to have the same settlement values as the original values
-
-    def equivalent_transfers(balance_proof):
```

Comment:
These mimicked balance proofs were removed, because they were not correct in the first place. There are additional constraints imposed on the locked amount, that should be `<=` total available deposit and `<=` than the available balance at a certain point in time. This is harder to mimic. Plus - they also took a lot of testing time (~10-15 min)

To replace them, I have added test values in `channel_settle_test_values` from `channel_test_values.py`. E.g.:
```
            # participant1 provides an old participant2 balance proof, but with the same
            # transferred + claimable_locked
            # This should have the same final participant balances as the valid last balance proofs
            ChannelValues(
                deposit=40,
                withdrawn=10,
                transferred=19994,
                claimable_locked=40,
                unclaimable_locked=2,
            ),
```